contents/blog/application-vs-infrastructure-open-source.md

```diff
@@ -0,0 +1,35 @@
+---
+date: 2021-01-01
+title: Why you should probably build an application layer open source startup
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: tim-glaser
+categories: ["CEO diaries"]
+---
+
+Alt. title: "Application vs infrastructe open source startups"
+Outline:
+- Defining application layer
+  - Examples of application layer: GitLab, HashiCorp?, Sentry, Mattermost  
+- Defining Infrastructure layer
+  - "Essentially SaaS apps that you can self host"
+  - Examples of infrastructure layer: Docker, Gatsby, TimescaleDB 
+- Differences, similarities
+- Why you should build an application layer open source startup
+  - Defensibility against AWS
+  - 'Mindset', ie people are used to paying for applications but not for infrastructure software. Linux is free
```

Comment:
This mindset angle is interesting.

I guess hosting becomes the only business model. ie MongoDB. However, it's definitely not a failure - I could see HN people picking up on this point.